compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1787,8 +1790,14 @@ impl<'tcx> TyCtxt<'tcx> {
     /// Determines whether an item is annotated with an attribute.
     pub fn has_attr(self, did: impl Into<DefId>, attr: Symbol) -> bool {
         let did: DefId = did.into();
-        if cfg!(debug_assertions) && !did.is_local() && rustc_feature::is_builtin_only_local(attr) {
-            bug!("tried to access the `only_local` attribute `{}` from an extern crate", attr);
+        if cfg!(debug_assertions)
+            && !did.is_local()
+            && !rustc_feature::is_builtin_encode_cross_crate(attr)
+        {
+            bug!(
+                "tried to access the `not encode cross crate` attribute `{}` from an extern crate",
+                attr
+            );
```

Comment:
this check is redundant :thinking: we check this in `get_attrs` already.

while you're touching this code, please remove it :grin: